### ISSUE TITLE
[loopring_v3,2,1][oedax_v1] lodash, silence dumb 'security' 

### ISSUE THREAD

**Issue by EvertonMelo** [Loopring/protocols#293]
# description:
You need other version lerna 3.13 for the 3.16. for the secure use: 

```
"devDependencies": {
    "lerna": "^3.13.1",
    "wsrun": "^3.6.4",
    "husky": "^1.3.1"
  }
}
```
according to the official website:
[lerna v3.16.0 - official repository](https://github.com/lerna/lerna/releases/tag/v3.16.0)

there is a bug in versions prior to:
` "lodash": "<4.17.14",` 

fixed https://github.com/lerna/lerna/commit/c405871 

## affected code:
```
"devDependencies": {
    "@types/bignumber.js": "^4.0.3",
    "@types/bluebird": "3.5.24",
    "@types/lodash": "^4.14.123",
    "@types/node": "9.3.0",
    "@types/request-promise-native": "^1.0.15",
    "@typescript-eslint/eslint-plugin": "^1.12.0",
    "@typescript-eslint/parser": "^1.12.0",
```
font: https://github.com/Loopring/protocols/blob/master/packages/loopring_v3/package.json#L47

```
devDependencies": {
    "@types/bignumber.js": "^4.0.3",
    "@types/bluebird": "^3.5.20",
    "@types/lodash": "^4.14.107",
    "@types/node": "9.3.0",
```
font: https://github.com/Loopring/protocols/blob/master/packages/loopring_v1/package.json#L29

 "devDependencies": {
    "@types/bignumber.js": "^4.0.3",
    "@types/bluebird": "3.5.24",
    "@types/lodash": "^4.14.107",
    "@types/node": "9.3.0",

font: https://github.com/Loopring/protocols/blob/master/packages/loopring_v2/package.json#L32

```
"devDependencies": {
    "@types/bignumber.js": "^4.0.3",
    "@types/bluebird": "3.5.24",
    "@types/lodash": "^4.14.123",
    "@types/node": "9.3.0",
```
font: https://github.com/Loopring/protocols/blob/master/packages/oedax_v1/package.json#L42

## Conclusion: 
according to the documentation of the correction, I suggest standardizing the lodash to avoid problems of insecure use and the upgrade of Lerna, so that the software does not have unpredictable results.

**Comment by dong77** [comment]
Thank you @EvertonMelo for submitting this issue.

**Comment by EvertonMelo** [comment]
hello  @dong77, thank you for accept my issues.

I'm participating in Bug Buounty protocols.

Could you confirm the severity? law or medium? 

it's 4 accumulated vulnerabilities right?

Do you have extra premium for those who fix the vulnerabilities?

**Comment by EvertonMelo** [comment]
upgrade lerna #299

**Comment by EvertonMelo** [comment]
loopring_v3 update lodash 4.17.14 

fixed [#300](https://github.com/Loopring/protocols/pull/300)

**Comment by EvertonMelo** [comment]
loopring_v1 update lodash 4.17.14

fixed [#301](https://github.com/Loopring/protocols/pull/301)

**Comment by EvertonMelo** [comment]
loopring_v2 update lodash 4.17.14

fixed [#302](https://github.com/Loopring/protocols/pull/302)

**Comment by EvertonMelo** [comment]
oedax_v1 update lodash 4.17.14 

fixed: [#303](https://github.com/Loopring/protocols/pull/303)
